test/check-storage-home-reuse

```diff
@@ -156,6 +174,16 @@ class TestStorageHomeReuseFedora(VirtInstallMachineCase):
 
     @disk_images([("fedora-rawhide", 15), ("fedora-41", 15)])
     def testHomeReuseFedoraEncrypted(self):
+        """
+        Description:
+            Test the home reuse installation method with a pre-existing standard
+            Fedora installation layout with encrypted /.
+
+        Expected results:
+            - The home reuse installation method is available
+            - The encrypted device needs to be unlocked
```

Comment:
I'd squash this into single
`- The home reuse installation method is available after the encrypted device is unlocked.`
I think we could update the test to check that it is not available before the unlocking.
`- The home reuse installation method is available only after the encrypted device is unlocked.`